packages/edit-site/src/components/global-styles/font-library-modal/font-collection.js

```diff
@@ -400,16 +403,26 @@ function PaginationFooter( { page, totalPages, setPage } ) {
 	);
 }
 
-function InstallFooter( { handleInstall, isDisabled } ) {
+function InstallFooter( { handleInstall, isDisabled, selectedFont } ) {
 	const { isInstalling } = useContext( FontLibraryContext );
 
+	const fontUploadsEnabled = useSelect( ( select ) => {
+		const { getEditorSettings } = select( editorStore );
+		return getEditorSettings().fontUploadsEnabled;
+	}, [] );
+
+	const isInstallButtonDisabled =
+		isDisabled ||
+		isInstalling ||
+		( ! fontUploadsEnabled && selectedFont?.fontFace?.length );
+
```

Comment:
@matiasbenedetto Since `InstallFooter` already has the `isDisabled` prop, would it be cleaner to move this logic into `FontCollection` and modify the prop, rather than override it here within the component?

Reply:
yes, sounds good

Reply:
Looks like this is a non-issue after merging from trunk, since the `InstallFooter` component no longer exists.